### PR TITLE
assorted fixes

### PR DESCRIPTION
**Pull request by mythi**
Fixes: #1284
Fixes: #1277

### CODE REVIEW

File: test/e2e/deviceplugins_suite_test.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func setupFirstNode() []byte {
 	utils.Kubectl("node-feature-discovery", "apply", "-k", "deployments/nfd/overlays/node-feature-rules/kustomization.yaml")
 
 	if err = e2epod.WaitForPodsRunningReady(c, "node-feature-discovery", 2, 0,
-		100*time.Second, map[string]string{}); err != nil {
+		200*time.Second, map[string]string{}); err != nil {
```
Comment (tkatila) [inline test/e2e/deviceplugins_suite_test.go:111]: Using proxy/mirror didn't fully fix the e2e issues?

**Review by tkatila — APPROVED** [review]
lgtm

File: test/e2e/deviceplugins_suite_test.go (hunk 1 above)
Comment (mythi) [reply test/e2e/deviceplugins_suite_test.go:111]: I'm preparing for another change that makes the spr/gnr instances to run a bit slower.

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**